zio-http-logging/src/test/scala/zhttp/logging/LoggerSpec.scala

```diff
@@ -0,0 +1,117 @@
+package zhttp.logging
+import zhttp.logging.LoggerTransport.Transport
+import zhttp.logging.LoggerTransport.Transport.unsafeSync
+import zio.test._
+
+import java.nio.file.{Files, Paths}
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+object LoggerSpec extends DefaultRunnableSpec {
+
+  private def imMemoryTransport(ref: mutable.ListBuffer[CharSequence]): Transport = unsafeSync { line =>
+    ref += line
+  }
+
+  private def inMemoryLogTransport(ref: mutable.ListBuffer[CharSequence]): LoggerTransport =
+    LoggerTransport(
+      format = LogFormat.min,
+      level = LogLevel.Info,
+      transport = imMemoryTransport(ref),
+    )
```

Comment:
I think we should reify the in memory implementation

```scala
final class MemoryLogger() extends Transport {
  val buffer: ListBuffer[String] = ListBuffer.empty[String]
  override def run(msg: CharSequence): Unit = buffer += msg.toString
  def stdout: String = buffer.mkString("\n")
}
```

Reply:
done